### DISCUSSION TITLE
Turning off RLV should turn off the Relay

### DISCUSSION THREAD

**Discussion by Slanador** [OpenCollarTeam/OpenCollar#1076] [Ideas]
Using version 8.2.x, I'm expecting that if I turn off RLV, that the Relay (which is at least from a user's perspective a part of the RLV feature) should also be turned off. Instead, the Relay stays on.

Three possible ways to improve:
a) Have the RLV Relay its own seperate menu item (maybe not ideal, it's a bit crowded)
b) When turning off RLV features, also turn off the Relay.
c) Have the Relay a separate item altogether, separate from the collar itself.

**Comment by Slanador** [comment]
Part of this is also user error: I wanted to turn off the Relay, but instead I turned off the RLV. I assumed that that would turn off the Relay also, because I wanted to use a multirelay instead, like DEM. In my defense, I think it's intuitive to assume that Relay is a subfeature of RLV.

**Comment by SilkieSabra** [comment]
I agree that it is intuitive to assume the relay is a sub feature of RLV.  The reasons it is not have to do with the role of the relay which is basically a benign parasitic life form in the collar.  It has its own system of permissions, so it can be triggered separately from collar access.  Turning off the relay when you turn off RLV would lose this feature.  We've tried having the relay as separate item linked to the collar similarly to an add-on.  That leaves moving the relay out of the RLV menu to its own link in the front page, In my opinion this might reduce some confusion but not sure it's worth it.  Anyone else have some input here?

**Comment by NikkiLacrima** [comment]
** edited **

Yes just looking at the menus, [CLEAR ALL] clears both collar RLV things and relay restrictions, nowhere is it stated that the [OFF] button is only for collar RLV stuff and not the relay, and then [Relay] goes to a submenu. And when the RLV is OFF the Relay submenu is not reachable and the CLEAR ALL not possible, even though the Relay is happily chugging along and locking the wearer to all kinds of evil furniture.

So is the Relay still running when RLV is set to off ? Formally yes it is running, still listening to incoming calls on the RLV_RELAY_CHANNEL, answering !commands and passing the @cmd rlv commands to oc_rlvsys but since RLV is turned off there  the RLV @commands are never sent to the viewer RLV subsystem and nothing happens. @version and @versionnew are therefore not answered and versionchecks will fail.
